codewof/static/js/question_types/base.js

```diff
@@ -25,6 +28,31 @@ function clear_submission_feedback() {
     $('#submission_feedback').empty();
 }
 
+function update_gamification(data) {
+    curr_points = data.curr_points;
+    $('#user_points_navbar').innerText = curr_points;
+    $("#user_points_navbar").load(location.href + " #user_points_navbar"); // Add space between URL and selector.
+
+    point_diff = parseInt(data.point_diff);
+    if(point_diff > 0) {
+        $("#point_toast_header").text("Earned " + point_diff.toString() + " points!");
+        $("#point_toast_body").text("Your total points are now " + data.curr_points.toString() +"!");
+        $(document).ready(function(){
+            $("#point_toast").toast('show');
+        });
```

Comment:
If a question doesn't fit the page (e.g a parsons) and you submit while scrolled down the box is never seen